autoPyTorch/pipeline/components/training/trainer/__init__.py

```diff
@@ -384,12 +395,6 @@ def early_stop_handler(self, X: Dict[str, Any]) -> bool:
             torch.save(X['network'].state_dict(), best_path)
 
         if epochs_since_best > X['early_stopping']:
```

Comment:
`return epochs_since_best > X['early_stopping']`

Reply:
Thanks for the suggestion, I have incorporated this.